backend/apps/volontulo/serializers.py

```diff
@@ -209,9 +209,9 @@ def get_is_administrator(obj):
     def get_organizations(self, obj):
         """Returns organizations that user belongs to."""
         qs = obj.userprofile.organizations.all()
-        return OrganizationSerializer(qs, many=True, context=self.context).data\
+        return OrganizationSerializer(qs, many=True, context=self.context).data
 
-    def get_phone_no(self, obj):
+    def get_phone_no(obj):
```

Comment:
Ok, I  see that You fixes few issues, but there are still two that requires attention:
1. Missing dosctring (remember, PEP-257: https://www.python.org/dev/peps/pep-0257/)
2. If You want to omit `self` (which is not used in `get_phone_no` function You need to inform Python about it using `staticmethod` decorator here, see: https://stackoverflow.com/a/12179752

You also have failing Travis build (https://travis-ci.org/CodeForPoznan/volontulo/builds/363874982) but it should be fixed when You will tacle these issues listed above.